### PR TITLE
Added DOKI

### PR DESCRIPTION
**Pull request by AddySiz**
## Description

 Adding DOKI from Odin chain

### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: ______
- [x] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [x] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -13707,7 +13707,49 @@
...
+      "name": "DOKI",
+      "display": "doki",
+      "symbol": "DOKI",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "odin",
+            "base_denom": "udoki",
+            "channel_id": "channel-3"
```
Comment (danbryan) [inline osmosis-1/osmosis-1.assetlist.json:13737]: /
```suggestion
            "channel_id": "channel-74"
```

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -13707,7 +13707,49 @@
...
+          },
+          "chain": {
+            "channel_id": "channel-258",
```
Comment (danbryan) [inline osmosis-1/osmosis-1.assetlist.json:13740]:
```suggestion
            "channel_id": "channel-20925",
```

File: osmosis-1/osmosis-1.assetlist.json [hunk 3]
```diff
@@ -13707,7 +13707,49 @@
...
+            "path": "transfer/channel-258/udoki"
```
Comment (danbryan) [inline osmosis-1/osmosis-1.assetlist.json:13741]:
```suggestion
            "path": "transfer/channel-20925/udoki"
```

**Review by danbryan — CHANGES_REQUESTED** [review]
updated channels for doki request

File: osmosis-1/osmosis-1.assetlist.json [hunk 4]
```diff
@@ -13707,7 +13707,49 @@
         "osmosis-price:ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4:1474",
         "peg:collateralized",
         "osmosis-unlisted"
+      ],
+       {
+      "description": "DOKI The last Dragon!🐲",
+      "denom_units": [
+        {
+          "denom": "ibc/C12C353A83CD1005FC38943410B894DBEC5F2ABC97FC12908F0FB03B970E8E1B",
```
Comment (danbryan) [inline osmosis-1/osmosis-1.assetlist.json:13715]:
```suggestion
          "denom": "ibc/BADB06C54ADD5BC4C8B74982F961CB0287BAE326E799FCD8D03387EB8BB7D550",
```

File: osmosis-1/osmosis.zone_assets.json [hunk 5]
```diff
@@ -2682,6 +2682,14 @@
...
       ],
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "odin",
+      "base_denom": "udoki",
+      "path": "transfer/channel-20925/udoki",
+      "twitter_URL": "https://twitter.com/doki_coin",
+      "osmosis_verified": false
+      ]
```
Comment (Satchitananda) [inline osmosis-1/osmosis.zone_assets.json]: @AddySiz remove this line

File: osmosis-1/osmosis.zone_assets.json [hunk 6]
```diff
@@ -2721,6 +2721,7 @@
       "osmosis_unlisted": true
     },
     {
+
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2724]: Please remove this extra line

File: osmosis-1/osmosis.zone_assets.json [hunk 7]
```diff
@@ -2806,6 +2807,13 @@
       "osmosis_verified": true,
       "osmosis_unlisted": true,
       "listing_date_time_utc": "2024-03-12T12:00:00Z"
+      },
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2810]: Please take out on indent here for proper formatting

File: osmosis-1/osmosis-1.assetlist.json [hunk 8]
```diff
@@ -14210,6 +14210,48 @@
         "osmosis-main",
         "osmosis-unlisted"
       ]
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:14214]: Please do not update the assetlist.json file directly

File: osmosis-1/osmosis.zone_assets.json [hunk 9]
```diff
@@ -2806,6 +2807,13 @@
       "osmosis_verified": true,
       "osmosis_unlisted": true,
       "listing_date_time_utc": "2024-03-12T12:00:00Z"
+    },
+    {
+      "chain_name": "odin",
+      "base_denom": "udoki",
+      "path": "transfer/channel-258/udoki",
+      "twitter_URL": "https://twitter.com/doki_coin",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2814]: Could you please Register this (and a website if there is one) to the Cosmos Chain Registry?
It is not required for listing, but it is become part of the 'Verified' status upgrade process.

**Comment by JeremyParish69** [comment]
Please update once Pool ID is known.

Also, see the latest updates on Verified process: https://github.com/osmosis-labs/assetlists/blob/main/LISTING.md

**Pull request merged**